<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hires</class>
 <widget class="QDialog" name="Hires">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>801</width>
     <height>561</height>
    </rect>
   </property>
   <property name="title">
    <string>Hires</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>92</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Go Back</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>771</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>518</width>
      <height>38</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="startDateLabel">
       <property name="text">
        <string>Start Date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="startDateLineEdit">
       <property name="text">
        <string>YYYY-MM-DD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="EndDateLabel">
       <property name="text">
        <string>End Date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="endDateLineEdit">
       <property name="text">
        <string>YYYY-MM-DD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="graphsRefreshButton">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>450</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Hires Data</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="allShopsLabel">
          <property name="text">
           <string>All Shops : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="shop1Label">
          <property name="text">
           <string>Shop 1 : </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="shop2Label">
          <property name="text">
           <string>Shop 2 : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="shop3Label">
          <property name="text">
           <string>Shop 3 : </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
